src/go/rdctl/pkg/config/config.go

```diff
@@ -51,26 +52,28 @@ type ConnectionInfo struct {
 var (
 	connectionSettings ConnectionInfo
 
-	configDir  string
 	configPath string
 	// DefaultConfigPath - used to differentiate not being able to find a user-specified config file from the default
 	DefaultConfigPath string
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
+	var configDir string
 	var err error
 	if runtime.GOOS == "linux" && isWSLDistro() {
 		if configDir, err = wslifyConfigDir(); err != nil {
 			log.Fatalf("Can't get WSL config-dir: %v", err)
 		}
+		configDir = filepath.Join(configDir, "rancher-desktop")
 	} else {
-		configDir, err = os.UserConfigDir()
+		appPaths, err := paths.GetPaths()
```

Comment:
In `cmd/shell.go` you are importing `paths.go` as `p` and use `paths` as the local variable. Here you import it by the normal name and use `appPaths` for the variable instead.

I think this should be handled consistently. I'm not a fan of using `p` for the import, so using `appPaths` seems better to me.

I would even go ahead and change `paths.GetPaths()` to `paths.GetAppPaths()` to match, but maybe that is too much for this PR?